tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -101,10 +102,17 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 		// decide which test setup is available to run
+		if vanillaCluster {
+			ginkgo.By("CNS_TEST: Running for Vanilla setup")
+			scParameters[scParamFsType] = ext4FSType
```

Comment:
why do you need this param and why only for vanilla?

Reply:
In case of vanilla and GC , SC parameter is PolicyID : scParameters[scParamStoragePolicyID] = profileID

Reply:
still confused but you don't need to set `scParamFsType`, from what you are saying you need to set
`scParameters[scParamStoragePolicyID] = profileID` here instead right

Reply:
You are correct. "scParamFsType" is not needed. Removed it.